### ISSUE TITLE
User can input the same task twice in a row.

### ISSUE THREAD

**Issue by GoogilyBoogily** [kyle-long/hackytrack#11]
This is probably not desired behavior.

**Comment by kyle-long** [comment]
Agreed. Although it doesn't hurt anything unless you are using `-vv` with `hackyanalyze`. I will consider this a low priority but it should be fixed at some point. Thanks
